src/servers/graphql-server.ts

```diff
@@ -84,6 +85,13 @@ const sessionContext = ({ token, ips, body, apiKey, apiSecret }) => {
             reason: loggedInUser,
           })
         domainUser = loggedInUser
+
+        const loggedInDomainAccount = await AccountsRepository().findById(
+          domainUser.defaultAccountId,
+        )
+        if (loggedInDomainAccount instanceof Error) throw Error
```

Comment:
Can probably throw some sort of `ApolloError` here with the `reason: loggedInDomainAccount` property